test/ductile/auth_test.clj

```diff
@@ -0,0 +1,44 @@
+(ns ductile.auth-test
+  (:require [ductile.auth :as sut]
+            [clojure.test :refer [deftest testing is]]))
+
+(deftest api-key-auth-test
+  (is (= {:headers
+          {:authorization
+           "ApiKey bmdrdkxuWUI0WmVoR1cxcVUtWHo6NkhNbkFDUFJTVldTTXZaQ2Y5VmNHZw=="}}
+         (sut/api-key-auth {:id "ngkvLnYB4ZehGW1qU-Xz"
+                            :api-key "6HMnACPRSVWSMvZCf9VcGg"}))))
+(deftest basic-auth-test
+  (is (= {:basic-auth ["the-login" "the-pwd"]}
+         (sut/basic-auth {:user "the-login" :pwd "the-pwd"}))))
+
+(deftest oauth-token-test
+  (is (= {:oauth-token "any-token"}
+         (sut/oauth-token {:token "any-token"}))))
+
+(deftest bearer-test
+  (is (= {:oauth-token "Bearer any-token"}
```

Comment:
Here you can also assert that "Bearer any-token" -> "Bearer any-token", i.e. existing `Bearer ` is omitted